### PR TITLE
P2266R3 Simpler implicit move

### LINKED ISSUE #5575: [2022-07 CWG Motion 10] P2266R3 Simpler implicit move
[P2266R3](https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2022/p2266r3.html)

### PR DESCRIPTION
**Pull request by jensmaurer**
Fixes #5575
Fixes cplusplus/papers#968

### CODE REVIEW

File: source/compatibility.tex [hunk 1]
```diff
@@ -42,6 +42,19 @@
 
 \rSec2[diff.cpp20.expr]{\ref{expr}: expressions}
 
+\diffref{expr.prim.id.unqual}
+\change
+Change move-eligible \grammarterm{id-expression}s from lvalues to xvalues.
+\rationale
+Simplify the rules for implicit move.
+\effect
+Valid \CppXX{} code that relies on a returned \grammarterm{id-expression}'s
```
Comment (JohelEGP) [inline source/compatibility.tex:51]:
```suggestion
Valid \CppXX{} code that relies on a returned \grammarterm{id-expression}
```

**Review by JohelEGP — CHANGES_REQUESTED** [review]

File: source/compatibility.tex (hunk 1 above)
Comment (jensmaurer) [reply source/compatibility.tex:51]: I think the existing text is fine English grammar, but I'll defer to the editor @tkoeppe for that.

Comment (tkoeppe) [reply source/compatibility.tex:51]: I agree, both are fine, and no reason to deviate from the approved wording.

Comment (JohelEGP) [reply source/compatibility.tex:51]: Is this supposed to imply "_id-expression_'s value category"?

Comment (tkoeppe) [reply source/compatibility.tex:51]: No, I think it's fine as is. "X's being a Y" is the same as "the fact that X is a Y", so you can think of this as "the id-expression is an lvalue", which seems fine - either "an expression is an lvalue" or "an expression's value category is 'lvalue'", but the former seems definitely fine, and common.

Comment (JohelEGP) [reply source/compatibility.tex:51]: I see. Thank you.

**Pull request merged**